docs/testing-guidelines/TestRoadmap.md

```diff
@@ -44,7 +44,7 @@ Running code coverage more often on full PowerShell is something that we should
 We currently run only those tests which are tagged `CI` excluding the tag `SLOW` as part of our continuous integration systems.
 This means roughly 1/3rd of our github tests are not being run on any regular schedule.
 In order to provide us with higher confidence in our code, we should be running *ALL* of our tests on a regular basis.
-We have recently added to `AppVeyor` running all of our tests on a daily basis, but are not yet running these tests on Linux/Mac via `Travis`, which should be done.
+We have recently added to `Azure DevOps` running all of our tests on a daily basis, but are not yet running these tests on Linux/macOS, which should be done.
```

Comment:
I guess this sentence is not valid anymore, since Windows, Linux and macOS tests are now available as part of daily builds as well as CI builds for PRs.

Reply:
Correct, this sentence can be deleted.

Reply:
done